lib/tsh/tsh.go

```diff
@@ -79,6 +79,7 @@ func (tsh Tsh) Bench(ctx context.Context, flags BenchFlags, userHost, command st
 	log.Debugf("Running %s", cmd)
 	outputBytes, err := cmd.CombinedOutput()
 	output := string(outputBytes)
+	log.Println(output)
```

Comment:
Debug leftover?

Reply:
Yeah, sorry
Still trying to figure out what's the issue with using the `tsh ssh/bench` commands
I was trying to understand if the error I got locally was the same as in the pipeline (it is!)